examples/db-templates/README.md

```diff
@@ -0,0 +1,95 @@
+OpenShift 3 Database Examples
+=============================
+
+This directory contains example JSON templates to deploy databases in OpenShift.
+They can be used to immediately instantiate a database and expose it as a
+service in the current project, or to add it as a template that can be later
+used from the Web Console or the CLI.
+
+The examples can also be tweaked to create your own templates.
+
+
+## Ephemeral x Persistent
+
+For each supported database, there are two template files.
+
+Files named `*-ephemeral-template.json` use
+"[emptyDir](https://docs.openshift.org/latest/dev_guide/volumes.html)" volumes
+for data storage, which means that data is lost after a pod restart.
+This is tolerable for experimenting, but not suitable for production use.
+
+The other templates, named `*-persistent-template.json`, use [persistent volume
+claims](https://docs.openshift.org/latest/architecture/additional_concepts/storage.html#persistent-volume-claims) to request persistent storage provided by [persistent volumes](https://docs.openshift.org/latest/architecture/additional_concepts/storage.html#persistent-volumes), that must
+have been created upfront.
+
+
+## Usage
+
+### Instantiating a new database service
+
+Use these instructions if you want to quickly deploy a new database service in
+your current project. Instantiate a new database service with this command:
+
+    $ oc new-app /path/to/template.json
+
+Replace `/path/to/template.json` with an appropriate path, that can be either a local path or an URL. Example:
+
+    $ oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json
```

Comment:
Yes, the output is from actually running the command. It works without "-f" because new-app tries to open the file/URL and parse it. What does not work is using a URL that does not point to a raw file, like:

```
$ # won't work
$ oc new-app https://github.com/openshift/origin/blob/master/examples/db-templates/mongodb-ephemeral-template.json
```

For local files, the "-f" is optional as well.